the-list-ui/src/request-functions.js

```diff
@@ -1,7 +1,23 @@
+const BE_URL = process.env.REACT_APP_BACKEND_URL;
+
 /**
  * @param {string} url
  */
-export async function getStudents(url) {
+export async function getStudents(url = BE_URL) {
   const response = await fetch(`${url}/students`);
   return response.json();
 }
+
+/**
+ * @param {string} url
+ * @param {string} name
+ */
+export async function postStudents(name, url = BE_URL) {
```

Comment:
Здесь `url` аргумент он не нужен совсем.

```suggestion
export async function postStudents(name) {
```